### PR TITLE
Enable report output to console in addition to writing to file

### PR DESCRIPTION
**Pull request by ansujohn**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -93,6 +93,13 @@ Default: `-1` *(means no limit)*
 
 Number of warnings to trigger non-zero exit code.
 
+### console
```
Comment (kevva) [inline readme.md]: I would rather call it `stdout`.

Comment (ansujohn) [inline readme.md]: @kevva Thanks. Renamed to stdout.

**Comment by mkrystek** [comment]
Hi.
Any chance this will be merged soon ?
We've recently thought, in our project, that it would be nice to see immediate output on console as well as xml output for our tools and voila, this PR was already here :)

File: readme.md [hunk 2]
```diff
@@ -93,6 +93,13 @@ Default: `-1` *(means no limit)*
 
 Number of warnings to trigger non-zero exit code.
 
+### stdout
```
Comment (sindresorhus) [inline readme.md]: Needs a better name. Something that makes it clear it's in a addition to writing the file. This makes it seem that it won't write to stdout at all, even when `outputFile` is not set.

File: tasks/eslint.js [hunk 3]
```diff
@@ -46,8 +46,13 @@ module.exports = grunt => {
 
 		if (opts.outputFile) {
 			grunt.file.write(opts.outputFile, output);
-		} else if (output) {
-			console.log(output);
+			console.log("Writing lint results to " + opts.outputFile);
```
Comment (sindresorhus) [inline tasks/eslint.js]: This is not needed. Grunt logs file writes when run in verbose mode.

**Comment by sindresorhus** [comment]
I don't really see why this option is needed at all. Can't we just change it to always print to stdout even when `outputFile` is set?

**Comment by bonesoul** [comment]
@sindresorhus I'm also outputting to a file but still want to see the errors on console too. can't we get this merged?

**Pull request closed without merging**